src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaActivityService.java

```diff
@@ -139,4 +143,43 @@ public void syncActivityLocations(DwSection dwSection, List<Activity> activities
 			}
 		}
 	}
+
+	@Override
+	public Activity createFromDwActivity(DwActivity dwActivity) {
+		Activity activity = new Activity();
+
+		ActivityType activityType = new ActivityType();
+		activityType.setActivityTypeCode(dwActivity.getSsrmeet_schd_code());
+
+		activity.setActivityTypeCode(activityType);
+
+		String rawStartTime = dwActivity.getSsrmeet_begin_time();
+
+		if (rawStartTime != null) {
```

Comment:
Add a comment to describe what's going on here, e.g. substring(2, 4), (0, 2), etc.